examples/Enzyme/Standalone/EnzymeVector.cpp

```diff
@@ -77,15 +81,15 @@ void dsquare(std::vector<double> x, std::vector<double> y, DenseMatrix& dy)
 int main()
 {
   // Vector and matrix declarations
-  constexpr int       N = 10;
+  constexpr size_t    N = 10;
   std::vector<double> x(N);
   std::vector<double> sq(N);
   DenseMatrix         dsq     = DenseMatrix(N, N);
   DenseMatrix         dsq_ref = DenseMatrix(N, N);
 
   // Random input values
-  srand(time(NULL));
-  for (int idx = 0; idx < x.size(); ++idx)
+  srand(static_cast<unsigned int>(time(NULL)));
+  for (size_t idx = 0; idx < x.size(); ++idx)
   {
     x[idx] = rand();
   }
```

Comment:
you could use [`std::generate`](https://cppreference.com/w/cpp/algorithm/generate.html) here. additionally, is there are reason we're using the c standard library random and not the [c++ standard library](https://cppreference.com/w/cpp/numeric/random.html) random number generation?